ads/google/doubleclick.js

```diff
@@ -164,6 +172,12 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
   if (gladeExperiment === GladeExperiment.GLADE_EXPERIMENT) {
     jsonParameters.gladeExp = '1';
   }
+  const expIds = data['experimentId'];
+  if (expIds) {
+    jsonParameters.gladeEids = jsonParameters.gladeEids ?
+        jsonParameters.gladeEids + ',' + expIds : expIds;
```

Comment:
Yup, my PR at https://github.com/ampproject/amphtml/pull/4688 runs before this which is why @glevitzky ensured to check if it exists yet and comma separate if it does.